resources/options_lookup.tsv

```diff
@@ -419,11 +419,11 @@ Insulation Floor	Ceiling R-19	ResStockArguments	floor_over_foundation_assembly_r
-Insulation Foundation Wall	"Wall R-10, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=10	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto																				
-Insulation Foundation Wall	"Wall R-15, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=15	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto																				
-Insulation Foundation Wall	"Wall R-13, Interior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=13	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto																				
+Insulation Foundation Wall	Uninsulated	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=0	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=0	rim_joist_assembly_r=3.4																			
+Insulation Foundation Wall	"Wall R-5, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=5	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=7.5																			
+Insulation Foundation Wall	"Wall R-10, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=10	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=9.8																			
+Insulation Foundation Wall	"Wall R-13, Interior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=13	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=11.3																			
+Insulation Foundation Wall	"Wall R-15, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=15	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=12.1																			

```

Comment:
@joseph-robertson Some of these options say "Exterior" but all of the options have `foundation_wall_insulation_location=interior`

Reply:
Changed to `=exterior` for "Exterior" options.